ocs_ci/deployment/aws.py

```diff
@@ -814,11 +814,11 @@ def destroy_cluster(self, log_level="DEBUG"):
         # Delete master, bootstrap, security group, and worker stacks
         suffixes = ['ma', 'bs', 'sg']
 
-        num_workers = config.ENV_DATA['worker_replicas']
-        for i in range(num_workers - 1, -1, -1):
-            suffixes.insert(0, f'no{i}')
-
-        stack_names = [f'{cluster_name}-{suffix}' for suffix in suffixes]
+        stack_names = self.aws.get_worker_stacks()
+        stack_names.sort()
+        stack_names.reverse()
+        map(stack_names.append, [f'{cluster_name}-{s}' for s in suffixes])
```

Comment:
Is this actually appending to `stack_names` for you? When I test this in the interpreter I just get a `map` object and the `stack_names` list remains unchanged. If it's not, we can easily do this with `extend` instead of `map()`.

```python
stack_names.extend([f'{cluster_name}-{s}' for s in suffixes])
```

Reply:
@clacroix12 yes , I missed it, we can make inplace replacement to reflect using 
```python
list(map(stack_names.append, [f'{cluster_name}-{s}' for s in suffixes]))
```
However what you have proposed is very easy and obvious, I couldn't think of that. I will use what you have suggested.

Reply:
done